.travis.yml

```diff
@@ -33,29 +34,31 @@ env:
 matrix:
   fast_finish: true
   include:
-    - php: '7.0'
+    - php: '7.1'
       env: TARGET=docs
-    - php: '7.0'
+    - php: '7.1'
       env: TARGET=lint
     - php: '5.6'
       env: COMPOSER_FLAGS="--prefer-lowest"
-    - php: '7.0'
+    - php: '7.1'
       env: SYMFONY=2.8.*
-    - php: '7.0'
+    - php: '7.1'
+      env: SYMFONY=3.0.*
```

Comment:
the first 3 lines in the build matrix come from the php versions at the top of the travis config file: https://travis-ci.org/sonata-project/SonataDoctrinePhpcrAdminBundle/builds/198934854